cmd/ncproxy/nodenetsvc/nodenetsvc.proto

```diff
@@ -7,35 +7,47 @@ service NodeNetworkService {
     rpc ConfigureContainerNetworking(ConfigureContainerNetworkingRequest) returns (ConfigureContainerNetworkingResponse);
     rpc PingNodeNetworkService(PingNodeNetworkServiceRequest) returns (PingNodeNetworkServiceResponse);
 }
- 
-enum RequestType { 
-    Setup = 0; 
-    Teardown = 1; 
-}
- 
+
 message ConfigureNetworkingRequest {
     string container_id = 1;
     RequestType request_type = 2; // Setup for AddNIC and Teardown for DeleteNIC
 }
 
-message ConfigureNetworkingResponse{}
+message ConfigureNetworkingResponse {}
 
 message PingNodeNetworkServiceRequest {
-    string pingRequestMessage = 1;
+    string ping_request_message = 1;
 }
 
 message PingNodeNetworkServiceResponse {
-    string pingResponseMessage = 1;
+    string ping_response_message = 1;
+}
+
+enum RequestType { 
+    Setup = 0;
+    Teardown = 1;
 }
 
-message ConfigureContainerNetworkingRequest{
+message ConfigureContainerNetworkingRequest {    
     RequestType request_type = 1;
     string container_id = 2;
-    string network_namespace = 3;
+    string network_namespace_id = 3;
```

Comment:
Were they ok with this? This seems like a breaking change

Reply:
This was a change on their internal proto. I just updated this file to theirs.

Reply:
Gotcha ok, all is good then 😬